cluster-up/cluster/kind/podpreset.sh

```diff
@@ -0,0 +1,48 @@
+ #!/usr/bin/env bash
+
+set -xe
+
+source ${KUBEVIRTCI_PATH}/cluster/kind/common.sh
+
+function pod_preset::enable_admission_plugin() {
```

Comment:
nit: it should reflect the filename, so the `_` is redundant.

Reply:
Done